lib/rollup.config.js

```diff
@@ -41,6 +41,7 @@ const globals = {
   '@material-ui/core/Tabs': 'material-ui.Tabs',
   '@material-ui/styles': 'material-ui.styles',
   '@material-ui/core/SvgIcon': 'material-ui.SvgIcon',
+  '@material-ui/core/styles/colorManipulator': 'material-ui.styles.colorManipulator',
```

Comment:
I think guys don't want to provide two level path imports as public api. Only `@material-ui/core/styles` is public api.